xamarin/Xamarin.Forms
# Add ability to set WebView ExecutionMode property (addresses #4720)

### Description of Change ###

Add platform specific to set the ExecutionMode on a WebView on UWP

When set as 'SeparateProcess', the memory allocated by the WebView itself is all handled in a sub-process, which ensures the main process never crashes from running out of memory here from doing things like opening and closing youtube every 5 seconds.  This behavior will likely crash when the WebView is in-process due to the huge amounts of memory that is required by that website.

This change is related to issue #4720, however I do not believe that issue is actually a bug.  This change does nothing to address a "memory leak" directly, and due to the addition of the new protected member, I wasn't sure if this is classed as an "API change" but thought it would be better to be on the safe side...

### Issues Resolved ### 
- partially fixes #4720 

### API Changes ###

Added:
```C#
        public static readonly BindableProperty ExecutionModeProperty = BindableProperty.Create("ExecutionMode", typeof(WebViewExecutionMode), typeof(WebView), WebViewExecutionMode.SameThread);
	public enum WebViewExecutionMode
	{
		SameThread = 0,
		SeparateThread = 1,
		SeparateProcess = 2
	}
```

Xamarin.Forms.Platform.UWP.WebViewRenderer
 - protected WebViewExecutionMode ExecutionMode
 
Changed:
 None

Removed:
 None 

### Platforms Affected ### 
- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Simply opening and closing a view that contains a WebView, pointed at something like youtube.com will cause huge memory usage which is not immediately reclaimed.  When the WebView is running in SeparateProcess mode, there is a child process that is created that handles the memory itself (see [ExecutionMode property](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.webview.executionmode?view=winrt-19041)

### PR Checklist ###
